libs/cluster/Server/ClusterConfig.cs

```diff
@@ -288,6 +288,26 @@ public List<int> GetLocalPrimarySlots()
             return slots;
         }
 
+        /// <summary>
+        /// Retrieve a list of slots served by this node's primary.
+        /// </summary>
+        /// <returns>List of slots.</returns>
+        public List<int> GetLocalSlots()
```

Comment:
I am confused about the purpose of this method. The description seems to indicate that you want to retrieve the slots served by this node's primary. However, you are initializing primaryId from this.LocalNodeId. Shouldn't be this LocalNodePrimaryId?

Reply:
My bad, I copied the comment over from the other method, it's meant to retrieve the slots for the local node.
I'll update.